packages/backend/src/dbt/translator.ts

```diff
@@ -48,9 +48,13 @@ const getDataTruncSql = (
     adapterType: SupportedDbtAdapter,
     timeInterval: string,
     field: string,
+    type: DimensionType,
 ) => {
     switch (adapterType) {
         case SupportedDbtAdapter.BIGQUERY:
+            if (type === DimensionType.TIMESTAMP) {
+                return `CAST(DATETIME_TRUNC('${field}', ${timeInterval.toUpperCase()}) as DATE)`;
```

Comment:
Do we need to cast it to a DATE ? won't it lose the time accuracy ?

Reply:
Looks like it I'm correct, so we can remove the cast function.
<img width="586" alt="Screenshot 2021-12-23 at 10 10 52" src="https://user-images.githubusercontent.com/9117144/147225319-3be0c5b1-e07a-44cd-a32b-2ef2e3f4dfbb.png">

Reply:
Well our time intervals are at a minimum day so we won't lose any info apart from timezone perhaps. I was trying to achieve a consistent output from the function but actually the other dbs all vary their output depending on input. I'll make a change.

Reply:
I already pushed a change to remove the cast. Let me know if you want to try something else. Otherwise I'm going to run e2e tests and merge.

Reply:
Ah ok was just making the changes on my branch I'll stop